src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -952,7 +952,7 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
                 }
                 else
                 {
-                    CopySelectionToClipboard(!shiftEnabled);
+                    CopySelectionToClipboard(shiftEnabled);

```

Comment:
Just commenting to make sure you're absolutely certain that you've plumbed this through correctly switching the flag to the opposite meaning.

Reply:
Verified as correct in the following cases:
- WT: mouse copy (w/ + w/out shift)
- WT: keyboard copy (w/ + w/out shift)
- ConHost: mouse copy (w/ + w/out shift)
- ConHost: keyboard copy (w/ + w/out shift)

All work as expected. :)